ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/execution/BaseJsonRpcProcessor.java

```diff
@@ -41,8 +41,10 @@ public JsonRpcResponse process(
     try {
       return method.response(request);
     } catch (final InvalidJsonRpcParameters e) {
+      JsonRpcError invalidParamsError = JsonRpcError.INVALID_PARAMS;
+      invalidParamsError.setData(e.getMessage());
```

Comment:
This doesn't touch the Engine API does it?
I think error messages are part of the spec in some cases for that: https://github.com/ethereum/execution-apis/blob/main/src/engine/common.md#errors